scss/6-components/_callout.scss

```diff
@@ -85,13 +89,52 @@
                 }
 
                 .cads-lang-gd &::before {
-                    content: 'Cudromach';
+                    content: 'Important';
                 }
 
                 .cads-lang-cy &::before {
-                    content: 'Bwysig';
+                    content: 'Important';
                 }
             }
         }
+
+        &adviser {
+            border-color: $cads-language__brand-adviser;
+            background-color: cads-colour-to-bg($cads-language__brand-adviser);
+
+            .cads-callout__adviser-label {
+                display: block;
+                /* stylelint-disable-next-line */
+                &::before {
+                    margin-bottom: $cads-spacing-3;
+                    background: $cads-language__brand-adviser;
+                    color: $cads-palette__white;
+                    padding: $cads-spacing-3;
+                    font-size: 0.875rem;
+                    font-weight: $cads-font-weight__bold;
+                    line-height: 1em;
```

Comment:
mixed rem and em for font-size and line-height makes my head explode. Can we drop the unit on line-height for similar effect?

Reply:
Done